pkg/controller/apiextensions/composite/reconciler.go

```diff
@@ -363,6 +363,13 @@ func (r *Reconciler) Reconcile(req reconcile.Request) (reconcile.Result, error)
 	refs := make([]corev1.ObjectReference, len(comp.Spec.Resources))
 	copy(refs, cr.GetResourceReferences())
 
+	// Inline PatchSets from Composition Spec before rendering
```

Comment:
@negz Wasn't 100% sure if this is where you meant by 
> the controller would call InlinePatchSets() then proceed as usual

But it works here as tested locally, and I'm not sure there's anywhere else that the `CompositionSpec` is available to have the method called on it 🙂

Reply:
This matches about what I was thinking.